app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1310,7 +1317,7 @@ public boolean onKeyDown(int keyCode, KeyEvent event) {
           } else {
             finish();
           }
-          return true;
```

Comment:
Why is this removed?

Reply:
Earlier, this method was giving the user a warning while leaving the `ZimHostActivity` that "the server will be stopped". But we're not stopping the server even though user is not in its activity or even the activity is destroyed. So this method was useless for us.

Reply:
@iadeelzafar I'm talking about the return statement in MainActivity or does it affect something in ZimHostActivity?

Reply:
@abdulwd This seems like a mistake. I'll redo it.

Reply:
Fixed. 54fbf85fc66132b546a939011b76b3318fb14154